include/quicr/detail/priority_queue.h

```diff
@@ -112,39 +112,39 @@ namespace quicr {
         /**
          * @brief Get the first object from queue
          *
+         * @param elem[out]          Time queue element storage. Will be updated.
+         *
          * @return TimeQueueElement<DataType> value from time queue
          */
-        TimeQueueElement<DataType> Front()
+        void Front(TimeQueueElement<DataType>& elem)
         {
             std::lock_guard<std::mutex> _(mutex_);
 
             for (auto& tqueue : queue_) {
                 if (!tqueue || tqueue->Empty())
                     continue;
 
-                return tqueue->Front();
+                tqueue->Front(elem);
             }
-
-            return {};
         }
 
         /**
          * @brief Get and remove the first object from queue
          *
+         * @param elem[out]          Time queue element storage. Will be updated.
+         *
          * @return TimeQueueElement<DataType> from time queue
          */
-        TimeQueueElement<DataType> PopFront()
+        void PopFront(TimeQueueElement<DataType>& elem)
         {
```

Comment:
same question as above

Reply:
Same as about,  but for this one the original is PopFront still available.